apps/vx-scan/backend/src/vx_interpreter.test.ts

```diff
@@ -294,16 +305,19 @@ test('interprets marks on an upside-down HMPB', async () => {
 
   const ballotImagePath = stateOfHamiltonFixtures.filledInPage1Flipped;
   expect(
-    (
-      await interpreter.interpretFile({
-        ballotImagePath,
-        detectQrcodeResult: await detectQrcodeInFilePath(ballotImagePath),
-      })
-    ).interpretation as InterpretedHmpbPage
+    replaceSnapshotValues(
+      (
+        await interpreter.interpretFile({
+          ballotImagePath,
+          detectQrcodeResult: await detectQrcodeInFilePath(ballotImagePath),
+        })
+      ).interpretation as InterpretedHmpbPage,
+      { expectedElectionHash: expect.anything() }
+    )
```

Comment:
My other concern with `replaceSnapshotValues` is that it will enable us to rely on snapshots more, rather than dissuading us from using snapshots (which I think is a better tendency overall)

Reply:
In this case it may be easier, but not in all cases. I am not always sure what the test is really covering, so I can't convert snapshots to assertions. I will replace snapshots in this PR with assertions where easier, but I don't think I can fully remove the need for `replaceSnapshotValues`. I agree with you that we should lean away from snapshots but I'm not concerned `replaceSnapshotValues` will make that harder.